pages/players/[id]/ranking.js

```diff
@@ -0,0 +1,97 @@
+import { useRouter } from "next/router";
+import { StyledLink, LinkText } from "../../../components/StyledLink";
+import styled from "styled-components";
+import useSWR from "swr";
+import Ranking from "../../../components/Ranking";
+
+export default function RankingPage() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { id } = router.query;
+  const { data: player, isLoading, error } = useSWR(`/api/players/${id}`);
+
+  if (!isReady || isLoading || error)
```

Comment:
The same is true here (`useEffect` in case you experience errors related to `isReady`).